.github/CODEOWNERS

```diff
@@ -2,11 +2,11 @@
 # These owners will be automatically assigned to any PRs
 # opened for vulnerabilities to be added to the database
 # of the npm community ecosystem
-/vuln/npm/    @nodejs/ecosystem-security
+/vuln/npm/    @nodejs/security-triage  
```

Comment:
I'm not sure that this one should be changing, based on looking at the members of these teams.

Reply:
I agree, but the former didn't work either. Or it's something else that got changed.

Reply:
Thing is. The auto-assign for the /processes dir works just fine

Reply:
Maybe there are issues with teams there? I also don't know a ton about the CODEOWNERS thing.